src/runner/main.cpp

```diff
@@ -82,7 +82,7 @@ void open_menu_from_another_instance(std::optional<std::string> settings_window)
 {
     const HWND hwnd_main = FindWindowW(L"PToyTrayIconWindow", nullptr);
     LPARAM msg = static_cast<LPARAM>(ESettingsWindowNames::Overview);
-    if (settings_window.has_value())
+    if (settings_window.has_value() && settings_window.value() != "")
     {
         msg = static_cast<LPARAM>(ESettingsWindowNames_from_string(settings_window.value()));
     }

```

Comment:
After `PostMessageW` I tried calling `SetForegroundWindow(hwnd_main)` however this did not bring the currently open instance to the foreground. This needs to be fixed before merging.